### DISCUSSION TITLE
Containers

### DISCUSSION THREAD

**Discussion by aloispichler** [i3/i3#5311] [Q&A] [answered]
I am working with i3 for years now.

What I see are the
* the monitors, and
* the windows.

However, always unseen and implicit, there are as well
* ‘containers’.

It seems to me that containers are a misconstruction. They overcomplicate window handling (involving parents and children, etc.). Qtile is simpler, because it refrains from using the concept of containers. I believe that containers discourage new users from using i3.

**Comment by Airblader** [comment, marked as answer]
I don't see a question here. If you like Qtile better, why not use it?
